src/components/StreamList.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import {
+  Flex,
+  Box,
+  Skeleton,
+  Text,
+} from '@chakra-ui/react';
+
+import { useInjectedProvider } from '../contexts/InjectedProviderContext';
```

Comment:
not a deal breaker, but we could reorder these imports based on the file structure in the style guide.